### DISCUSSION TITLE
Yeni sayfa da cüzdan ekleme

### DISCUSSION THREAD

**Discussion by Ebrulirem** [IBAX-io/go-ibax#471] [Q&A] [answered]
### Clear and concise description of the problem

Yeni yapılan sayfada cüzdan ekleme bulamıyoruz site birazdaha basit olabilir bence Windows olarak kullanım kolay gıbı ama Android de cok zor

### Suggested solution

Yeni yapılan sayfada cüzdan ekleme bulamıyoruz site birazdaha basit olabilir bence Windows olarak kullanım kolay gıbı ama Android de cok zor

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by HalloKyle** [comment, marked as answer]
Weaver is just a development tool, there is already a development team working on the IBAX wallet (both desktop and mobile versions), please be patient.
